apps/meteor/app/api/server/v1/users.ts

```diff
@@ -634,7 +635,22 @@ API.v1.addRoute(
 	{
 		async post() {
 			const { email } = this.bodyParams;
-			await sendWelcomeEmail(email);
+
+			if (!isSMTPConfigured()) {
+				throw new Meteor.Error('error-email-send-failed', 'SMTP is not configured', {
+					method: 'sendWelcomeEmail',
+				});
+			}
+
+			const user = await Users.findOneByEmailAddress(email.trim(), { projection: { name: 1 } });
+
+			if (!user) {
+				throw new Meteor.Error('error-invalid-user', 'Invalid user', {
```

Comment:
Can we use Error instead of Meteor.Error? We can use this opportunity to take one step to move away from Meteor.
If we do so, we might have to change the tests if there are any.

Reply:
take a look on this folder: `apps/meteor/lib/errors` @tiagoevanp

Reply:
The guideline works roughly like this:

If the error needs to be detailed and sent back to the **client/response**, we use `MeteorError` (without the dot).

If the error is **internal** and only meant to break the execution flow, we use the standard `Error`.

In short: `MeteorError` is for user-facing errors. Otherwise, always use `Error`.

Reply:
I think we can work that decisions better in other place than here. For now I think it's acceptable for the fix what I did.